src/MultiSelect/index.js

```diff
@@ -191,22 +190,20 @@ const MultiSelect = ({
         id={name}
         value={fieldValue || selectedItems.map(itemToString).join(",")}
       />
-      <DropdownTrigger
-        disabled={disabled}
-        isOpen={isOpen}
-        labelText={label}
-        startContent={renderTokens()}
-        errorText={errorText}
-        labelProps={{ ...getLabelProps() }}
-        {...getToggleButtonProps(
-          getDropdownProps({
-            ...triggerProps,
-          }),
-        )}
-        style={{
-          ...triggerProps.style,
-        }}
-      />
+      <div {...triggerProps}>
+        <DropdownTrigger
+          disabled={disabled}
+          isOpen={isOpen}
+          labelText={label}
+          startContent={renderTokens()}
+          errorText={errorText}
+          labelProps={{ ...getLabelProps() }}
+          {...getToggleButtonProps()}
```

Comment:
This prop getter is returned by `useSelect`, which manages the state of the select popover.

The `useMultipleSelection` hook only needs to be responsible for managing the state of selected items, so I was able to remove the `getDropdownProps` getter. The `getDropdownProps` getter is only necessary if `useMultipleSelection` is being used in isolation (without `useSelect`).